spec/integration_spec.rb

```diff
@@ -155,4 +155,23 @@ def cleanup(output)
         .to match(/^local_oclc/)
     end
   end
+
+  describe "Scrub" do
+    it "'scrub x' loads records and produces output for org x" do
+      # Needs a bit of setup.
+      local_d = "/tmp/local_member_data/umich-hathitrust-member-data/print\ holdings/#{Time.new.year}/"
+      remote_d = "/tmp/remote_member_data/umich-hathitrust-member-data/print\ holdings/#{Time.new.year}/"
+      logfile = File.join(remote_d, "umich_mono_#{Date.today}.log")
+      FileUtils.rm_rf(local_d)
+      FileUtils.mkdir_p(remote_d)
+      FileUtils.touch("/tmp/rclone.conf")
+      FileUtils.rm_rf(logfile)
+      FileUtils.cp("spec/fixtures/umich_mono_full_20220101.tsv", remote_d)
+      # Actual tests:
+      expect(File.exist?(File.join(remote_d, "umich_mono_#{Date.today}.log"))).to be false
```

Comment:
This is just verifying a precondition, right? Might be worth putting that before the "actual tests" comment to clarify that nothing has happened yet